lib/mdi/Makefile.mpi

```diff
@@ -0,0 +1,30 @@
+# *
+# *_________________________________________________________________________*
+# *      Fortran Library for Reactive Force Field                           *
+# *      DESCRIPTION: SEE READ-ME                                           *
+# *      FILE NAME: Makefile                                                *
+# *      CONTRIBUTING AUTHORS: Hansohl Cho(MIT), Aidan Thompson(SNL)        *
```

Comment:
These comments seem incorrect.

Reply:
They were not correct have been removed.